src/rastervision/ml_backends/keras_classification.py

```diff
@@ -78,14 +78,31 @@ def _upload(data_uri):
 class ModelFiles(FileGroup):
     """Utilities for files produced when calling train."""
 
-    def __init__(self, base_uri):
+    def __init__(self, base_uri, replace_model=False):
```

Comment:
As a user, how does one specify that replace_model should be False, and that you want to continue training the model based on the checkpoint and log?

Reply:
Ah, I see - so if there is a checkpoint already, and you've either increased the number of epochs to train or the training was interrupted,  then keeping the model and log will let you continue from the last train point.

I hadn't considered this, let me think through this use case and enable it.

Reply:
Updated to add a train config option that allows the user to set this.